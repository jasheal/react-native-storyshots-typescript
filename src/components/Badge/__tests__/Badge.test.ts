import initStoryshots from '@storybook/addon-storyshots';

initStoryshots({
  storyKindRegex: /^Badge$/,
});
