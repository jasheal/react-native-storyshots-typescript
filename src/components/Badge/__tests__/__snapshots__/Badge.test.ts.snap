// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Badge ten 1`] = `
<View
  style={
    Object {
      "alignSelf": "center",
      "backgroundColor": "red",
      "borderRadius": 20,
      "marginTop": 50,
      "padding": 10,
    }
  }
>
  <Text
    style={
      Object {
        "color": "white",
        "fontWeight": "bold",
      }
    }
  >
    10
  </Text>
</View>
`;

exports[`Storyshots Badge three 1`] = `
<View
  style={
    Object {
      "alignSelf": "center",
      "backgroundColor": "red",
      "borderRadius": 20,
      "marginTop": 50,
      "padding": 10,
    }
  }
>
  <Text
    style={
      Object {
        "color": "white",
        "fontWeight": "bold",
      }
    }
  >
    3
  </Text>
</View>
`;

exports[`Storyshots Badge twenty 1`] = `
<View
  style={
    Object {
      "alignSelf": "center",
      "backgroundColor": "red",
      "borderRadius": 20,
      "marginTop": 50,
      "padding": 10,
    }
  }
>
  <Text
    style={
      Object {
        "color": "white",
        "fontWeight": "bold",
      }
    }
  >
    20
  </Text>
</View>
`;

exports[`Storyshots Badge two 1`] = `
<View
  style={
    Object {
      "alignSelf": "center",
      "backgroundColor": "red",
      "borderRadius": 20,
      "marginTop": 50,
      "padding": 10,
    }
  }
>
  <Text
    style={
      Object {
        "color": "white",
        "fontWeight": "bold",
      }
    }
  >
    2
  </Text>
</View>
`;
